### ISSUE TITLE
Instructions and/or scripts to use CI docker image for testing

### ISSUE THREAD

**Issue by gonuke** [svalinn/DAGMC#681]
For some contributors, the easiest way to test changes will be to use the docker image from CI.  However, our CI system is non-trivial, so it's not simple to use this approach.  We could add documentation for how to do this or add scripts to the CI or other folders to allow users to launch testing in the CI docker environment.

**Comment by bam241** [comment]
I have a script that does that, I could change it a little to allow people to run it locally....

**Comment by gonuke** [comment]
This is perhaps simpler than I first thought.  I've added comments to the `install.sh` and `tests.sh` scripts for guidance as part of #687

**Comment by gonuke** [comment]
Captured from now-closed #687 - needs to be updated for new GHCR hosted docker images
```
# Manual testing that mimics CI
# --------------------------------
# 
# Run `CI/circleci/install.sh` in one of the CI Docker images with the root dir 
# of the DAGMC repo as the working directory.  Note that CI only runs on 
# MOAB_VERSION=5.1.0 and HDF5_VERSION=1.10.4
#
# If you are in the DAGMC root directory:
#
#    $ docker run -v ${PWD}:/root/build_dir/DAGMC -w /root/build_dir/DAGMC \
#              -it svalinn/dagmc-ci-ubuntu-16.04-clang-ext-hdf5_1.10.4-moab_5.1.0
#
# Inside the docker container:
#
#    $ CI/circleci/install.sh
#
#    $ CI/circleci/tests.sh
#
```

**Comment by ahnaf-tahmid-chowdhury** [comment]
I think we can close this issue.
